libseccomp/pkg.yaml

```diff
@@ -0,0 +1,29 @@
+name: libseccomp
+dependencies:
+  - stage: base
+steps:
```

Comment:
this is somewhat duplicate with `pkgs/`, but here it is consumed in static form, so it won't pollute the toolchain